webpack.config.js

```diff
@@ -89,6 +90,17 @@ const webpackConfig = {
                         ]
                     }
                 }
+            },
```

Comment:
We can use the same loader for js and ts files. This allows us to share preset, plugin an exclude setting both script types:

https://github.com/jwplayer/jwplayer-commercial/pull/7324/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR99-R120